### ISSUE TITLE
Blank and triangle on screen bug

### ISSUE THREAD

**Issue by Norbtails97** [kootstra-rene/enge-js#44]
---HELP WANTED--

Every time I load a rom I get this screen
![image](https://user-images.githubusercontent.com/123211390/222222581-f02c3885-b349-42fe-80ed-277da8fd190a.png)
The triangle is white on the boot and blue on the bios. are there's a way to fix  @kootstra-rene #

**Comment by kootstra-rene** [comment]
did you load a bios already? and which game?

**Comment by Norbtails97** [comment]
I LOADED THE BIOS AND I TRIED MULTIPLE ROMS BUT IT DID NOT WORK :(

**Comment by kootstra-rene** [comment]
hmm.. can you tell which bios? and what the extension of the roms are? the emulator does not support all formats yet. 

it could also be a hardware issue. on what kind of system are you trying to run the emulator?

**Comment by Norbtails97** [comment]
scph1001.bin. chd, bin, iso. And  on a school chromebook.

**Comment by kootstra-rene** [comment]
bin and iso should work but a chromebook I have never tried. do you know how to open the devtools of chrome? and if so can you send the me the text of the console-tab? or check if there are errors?

**Comment by Norbtails97** [comment]
devtools is blocked by the school

**Comment by kootstra-rene** [comment]
hmm, then it will be difficult to find out what is wrong. likely it is an issue in the shaders. meaning that the emulator uses some functionality that is unavailable on the chromebook, however without a chromebook this will be challenge to find out. I'll see if I can get my hands on a chromebook to test. Sorry can't help you now but maybe later.

can you tell me which chromebook you have?

**Comment by Norbtails97** [comment]
oh and one more thing. it worked on my chromebook a few weeks ago now it doesn't now.

**Comment by Norbtails97** [comment]
HP Chromebook 11MK G9 EE

**Comment by Norbtails97** [comment]
i found a way to unbock it this what it looks like
[Screen recording 2023-03-01 11.38.39 AM.webm](https://user-images.githubusercontent.com/123211390/222247171-bd17127d-1af5-4521-847d-61e7d8c899f1.webm)

**Comment by kootstra-rene** [comment]
Thanks for the info and video. I cant see an error. Did you do a complete reboot of the chromebook already?
can you type chrome://gpu/ in the address bar? you should see a lot info about the graphics card. 
maybe make video of that info?

**Comment by Norbtails97** [comment]
[Screen recording 2023-03-01 12.00.07 PM.webm](https://user-images.githubusercontent.com/123211390/222252673-c5236c62-c47c-4aa8-88cb-011cdacfccc0.webm)
complete reboot? ya mean powrwashing or restart?

**Comment by kootstra-rene** [comment]
actual turn-off -> turn on cycle.

**Comment by kootstra-rene** [comment]
looking at the info. it should work. #puzzling

**Comment by Norbtails97** [comment]
i realized it works but only video/images. but not 3d graphics

**Comment by kootstra-rene** [comment]
ah, someone else also had that issue.  but it remains weird that it used to work a few weeks ago. or was that also just 2d and video.

**Comment by Norbtails97** [comment]
Probably it's the updates on most web browsers. I hope you fix that soon!

**Comment by Norbtails97** [comment]
Are you working on the bug? i want to play crash bandicoot. :l

**Comment by kootstra-rene** [comment]
Well the emulator has low priority now because I have a job with two demanding projects and am building a house in the hours that are left when I am not sleeping... and I don't have a chromebook.

**Comment by Norbtails97** [comment]
ok. I also did some experiments today. I used old versions of browsers and it seems working. but on my school Chromebook. same problem.

**Comment by Norbtails97** [comment]
HEY! IT WORKS NOW!!! THANK YOU!!!! :D
![Screenshot 2023-04-21 9 41 25 AM](https://user-images.githubusercontent.com/123211390/233690045-c2ade5d3-bd8b-4b72-872d-6cb10b5471d7.png)

**Comment by kootstra-rene** [comment]
your welcome , now lets hope that the driver guys stop making incompatible changes..
